BenadrylAutomation/Configurations/config.properties

```diff
@@ -1,5 +1,5 @@
-stage_url=http://test.benadryl.com/
+stage_url=http://app-23191.on-aptible.com/
```

Comment:
@zahinSJI what are you doing man? this url is from other project. Why did u changed this ? and added here??

Reply:
![image](https://github.com/sji-autobots/BenadrylAutomation/assets/107922907/c1c0fec8-59a0-44fc-be89-25011847e6e2)

It is absolutely correct in my end, I haven't made any change to the file other than making it headless

Reply:
@zahinSJI can you please commit this file again?
i cannot see the change here.